README.md

```diff
@@ -1,29 +1,29 @@
 
 | **model**                    | **description**                                                                                                                                                 |
 | ---------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| zendesk_ticket_metrics       | Each record represents a Zendesk ticket, enhriched with metrics about reply times, resolution times and work times.  Calendar and business hours are supported  |
-| zendesk_ticket_enhanced      | Each record represents a Zendesk ticket, enriched with data about it's tags, assignees, requester, submitter, organization and group.                           |
-| zendesk_ticket_field_history | A daily historical view of the ticket field values defined in the ticket_field_history_columns variable                                                         |
+| zendesk_ticket_metrics       | Each record represents a Zendesk ticket, enhriched with metrics about reply times, resolution times, and work times.  Calendar and business hours are supported.  |
+| zendesk_ticket_enhanced      | Each record represents a Zendesk ticket, enriched with data about its tags, assignees, requester, submitter, organization, and group.                           |
+| zendesk_ticket_field_history | A daily historical view of the ticket field values defined in the `ticket_field_history_columns` variable .                                                        |
 | zendesk_sla_breach           | Each record represents an SLA breach event. Calendar and business hour SLA breaches are supported.                                                              |
 
 ## Installation Instructions
 Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions, or [read the docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
 
 ## Configuration
-By default this package will look for your Zendesk data in the `zendesk` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). If this is not where your Zendesk data is (perhaps your Zendesk schema is `zendesk_fivetran`), please add the following configuration to your `dbt_project.yml` file:
+By default, this package looks for your Zendesk Support data in the `zendesk` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). This might not be where your Zendesk Support data is (for example, your Zendesk Support schema may be called `zendesk_fivetran` instead). If your Zendesk Support schema is not called `zendesk`, add the following configuration to your `dbt_project.yml` file:
```

Comment:
@fivetran-kristin We specifically called out `zendesk_fivetran` because it's a common schema name, correct? If not, I'm inclined to remove it - it doesn't add a lot of meaning _unless_ it points out a common situation that customers will encounter.

Reply:
By far the most common schema name is `zendesk` as it is the default when you go through the connector set up process. `zendesk_fivetran` is not nearly as popular, but we do see people edit the default name to something else (which commonly includes `fivetran`).  

I'm totally fine with removing the example!

Reply:
Great! I'll remove it then. None of our other dbt package READMEs have them, and I don't think they add meaningful information.